### DISCUSSION TITLE
.gitattributes not working

### DISCUSSION THREAD

**Discussion by EmirYilmaz1** [github-linguist/linguist#6478] [Q&A] [answered]
Hi, my Unity game repository showing 2 Irrelevant language 
![Ekran görüntüsü 2023-07-14 154444](https://github.com/github-linguist/linguist/assets/78986122/c1a6ef63-77d2-4469-a8bf-99232c63ea94)

Even i add this line to my .gitattributes file. Where am i doing wrong? Can you please help me because its really annoying.
![Ekran görüntüsü 2023-07-15 134253](https://github.com/github-linguist/linguist/assets/78986122/eb55cd40-1990-496e-b24f-04facb78b88a)

This is my repository:
[https://github.com/EmirYilmaz1/Flappy-Bird-Clone](url)

**Comment by lildude** [comment, marked as answer]
The path is case-sensitive, so you should be using `**/Shaders/*`
